Client/src/Components/TabPanels/Status/GeneralSettingsPanel.jsx

```
@@ -0,0 +1,281 @@
+	const handleChange = (event) => {
+		event.preventDefault();
+		const { value, id } = event.target;
+		setLocalData((prev) => ({
+			...prev,
+			[id]: value,
+		}));
+	};
+
+	const handleSubmit = () => {
+		//validate rest of the form
+		delete localData.logo;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)